views/surveys/README.md

```diff
@@ -0,0 +1,32 @@
+# Surveys
+
+* Surveys are hosted on Google Drive using a Google Form which then sends the submitted form data to a linked Google Spreadsheet. 
+<<<<<<< HEAD
```

Comment:
`git merge` artifacts ln 4, 6, 32. Message me if you need help with git to avoid this.